mappings/net/minecraft/container/BeaconContainer.mapping

```diff
@@ -8,6 +8,10 @@ CLASS axu net/minecraft/container/BeaconContainer
 		ARG 1 syncId
 	METHOD <init> (ILagn;Laya;Layb;)V
 		ARG 1 syncId
+	METHOD c setEffects (II)V
+		ARG 1 primaryEffectId
+		ARG 2 secondaryEffectId
+	METHOD e getProperty ()I
```

Comment:
`getProperty`? What is the property?

Reply:
Given this is an integer, it looks like a flags field getter to me

Reply:
I imagine should at least be `getProperties` then?

Reply:
I named it `getProperty` because of the `setProperty` method in the same class. Should I rename both so they are plural?

Reply:
Huh, I might be missing it but I don't see a `setProperty` there. Is it in a superclass?

Reply:
yeah actually it's a method in `Container` class. Should still be plural though